pkg/controller/migration/convert/network.go

```diff
@@ -198,11 +198,11 @@ func handleCalicoCNI(c *components, install *operatorv1.Installation) error {
 		install.Spec.CalicoNetwork.HostPorts = &hp
 	}
 
-	if c.cni.ConfigName != "k8s-pod-network" {
+	if c.cni.ConfigName != "calico" {
```

Comment:
I think this change is going to break a lot of clusters.

Anyone using older versions of Calico installed via manifest will have `k8s-pod-network` set as the name, so we need to continue to support upgrading them.

Reply:
I think it should just be a matter of supporting both values here, instead of only matching on "calico"?